pkg/skaffold/deploy/kubectl/cli.go

```diff
@@ -62,12 +63,13 @@ type Config interface {
 	PipelineForImage(imageName string) (latestV1.Pipeline, bool)
 }
 
-func NewCLI(cfg Config, flags latestV1.KubectlFlags, defaultNamespace string) CLI {
+func NewCLI(cfg Config, flags latestV1.KubectlFlags, defaultNamespace string, dryRun string) CLI {
```

Comment:
Add some doc here.  If dryRun == "" then uses "client".

Reply:
Will do. I'll also move the empty string check to here rather than the ReadManifests function. That way it's already set to "client" in the case that the value ever gets read anywhere else.